guardrails-scripts/yb-voyager-pg-grant-migration-permissions.sql

```diff
@@ -1,83 +1,86 @@
 --- How to use the script:
 -- Run the script with psql command line tool, passing the necessary parameters:
---- psql -h <host> -d <database> -U <username> -v voyager_user='<voyager_user>' -v schema_list='<schema_list>' -v replication_group='<replication_group>' -v original_owner_of_tables='<original_owner_of_tables>' -v is_live_migration=<is_live_migration> -v is_live_migration_fall_back=<is_live_migration_fall_back> -f <path_to_script> 
+--- psql -h <host> -d <database> -U <username> -v voyager_user='<voyager_user>' -v schema_list='<schema_list>' -v is_live_migration=<is_live_migration> -v is_live_migration_fall_back=<is_live_migration_fall_back> -v replication_group='<replication_group>' -v original_owner_of_tables='<original_owner_of_tables>' -f <path_to_script> 
 --- Example:
---- psql -h <host> -d <database> -U <username> -v voyager_user='ybvoyager' -v schema_list='schema1,public,schema2' -v replication_group='replication_group' -v original_owner_of_tables='postgres' -v is_live_migration=1 -v is_live_migration_fall_back=0 -f /home/ubuntu/yb-voyager-pg-grant-migration-permissions.sql
+--- psql -h <host> -d <database> -U <username> -v voyager_user='ybvoyager' -v schema_list='schema1,public,schema2' -v is_live_migration=1 -v is_live_migration_fall_back=0 -v replication_group='replication_group' -v original_owner_of_tables='postgres' -f /home/ubuntu/yb-voyager-pg-grant-migration-permissions.sql
 --- Parameters:
 --- <host>: The hostname of the PostgreSQL server.
 --- <database>: The name of the database to connect to.
 --- <username>: The username to connect with.
 --- <voyager_user>: The database user for which permissions are being granted.
 --- <schema_list>: A comma-separated list of schemas to grant permissions on. Example 'schema1,public,schema2'.
+--- <is_live_migration>: A flag indicating if this is a live migration (1 for true, 0 for false). If set to 0 then the script will check for permissions for an offline migration.
+--- <is_live_migration_fall_back>: A flag indicating if this is a live migration with fallback (1 for true, 0 for false). If set to 0 then the script will detect permissions for live migration with fall-forward. Should only be set to 1 when is_live_migration is also set to 1. Does not need to be provided unless is_live_migration is set to 1.
 --- <replication_group>: The name of the replication group to be created. Not needed for offline migration.
 --- <original_owner_of_tables>: The original owner of the tables to be added to the replication group. Not needed for offline migration.
```

Comment:
Why do we need `original_owner_of_tables`. We can figure that out by ourselves, no?